src/Compilers/CSharp/Portable/Binder/InMethodBinder.cs

```diff
@@ -16,9 +16,9 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// </summary>
     internal sealed class InMethodBinder : LocalScopeBinder
     {
-        private readonly MultiDictionary<string, ParameterSymbol> _parameterMap;
+        private MultiDictionary<string, ParameterSymbol> _lazyParameterMap;
```

Comment:
Can you please explain how this contributes to the handling of the switch statement?